README.md

```diff
@@ -3,7 +3,7 @@
 
 </div>
 
-<!-- TABLE OF CONTENTS -->
+<!-- TABLE OF CONTENT -->
```

Comment:
- [ ] **Required:** As suggested by the previous reviewer, could you acknowledge the designer as this is required for resources not originally created by us that we use in our work? Give appropriate credit in this branch as well to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).